rfc6962-bis.xml

```diff
@@ -1201,6 +1201,9 @@ an accepted root certificate.
                     <t hangText="extra_data:">
                       The base64-encoded unsigned data pertaining to the log entry. In the case of an X509ChainEntry, this is the <spanx style="verb">certificate_chain</spanx>. In the case of a PrecertChainEntryV2, this is the whole <spanx style="verb">PrecertChainEntryV2</spanx>.
                     </t>
+                    <t hangText="signed_certificate_timestamp:">
+                      A Signed Certificate Timestamp for the entry, in the same format as in <xref target="sct"/>.
```

Comment:
OK.  In that case, "the same format" will become valid when we make that change.  :-)

Shall we also look at pruning the contents of "leaf_input" so that get-entries doesn't return any duplicated SCT fields?  I think we only need...
           LogEntryType entry_type;
           select(entry_type) {
               case x509_entry: ASN.1Cert;
               case precert_entry_V2: TBSCertificate;
           }

"LogEntryType entry_type" could become a field inside the SCT, as per TRAC ticket 10.